packages/react/src/primitives/Field/FieldClearButton.tsx

```diff
@@ -3,16 +3,20 @@ import * as React from 'react';
 import { FieldClearButtonProps, Primitive } from '../types';
 import { FieldGroupIconButton } from '../FieldGroupIcon';
 import { IconClose } from '../Icon';
-import { SharedText } from '../shared/i18n';
+import { SharedText } from '../shared/constants';
 
 const ariaLabelText = SharedText.Fields.ariaLabel.clearField;
 
 const FieldClearButtonPrimitive: Primitive<FieldClearButtonProps, 'button'> = (
-  props,
+  { ariaLabel, size, ...rest },
```

Comment:
Nit: should we do `ariaLabel = SharedText.Fields.ariaLabel.clearField` here like you do above in the Collection component?